controllers/release/adapter.go

```diff
@@ -373,6 +388,45 @@ func (a *adapter) createManagedPipelineRun(resources *loader.ProcessingResources
 	return pipelineRun, nil
 }
 
+// createTenantPipelineRun creates and returns a new tenant Release PipelineRun. The new PipelineRun will include owner
+// annotations, so it triggers Release reconciles whenever it changes. The Pipeline information and the parameters to it
+// will be extracted from the given ReleaseStrategy. The Release's Snapshot will also be passed to the release
```

Comment:
We don't have ReleaseStrategy anymore

Reply:
copy&paste. This is probably there also for the managed one. I will remove from both.

Reply:
Thanks!